src/util/download.js

```diff
@@ -0,0 +1,48 @@
+import { logWarning } from './logger.js';
+
+const REGEX_URL = /^(http(s?):)([\/|.|\w|\s\:\#|-])*\.(?:jpg|gif|png|fbx|glb)/;
+
+export function isUrl(url) {
+  return REGEX_URL.test(url);
+}
+
+export async function download (urlAddress, writablePath) {
+    // Validate input parameters
+    if (writablePath === undefined || writablePath === undefined || writablePath === '') {
```

Comment:
`writablePath === undefined` is used twice

Reply:
:+1: Thank you @pleszkiewicz !